lib/connection.rb

```diff
@@ -37,16 +38,17 @@ def options=(options)
     #
     def default_middleware
       Proc.new do |builder|
-        builder.use Faraday::Request::Multipart
+        builder.use Faraday::Multipart::Middleware
         builder.use Faraday::Request::UrlEncoded
         if configuration.authenticated?
-          builder.use Faraday::Request::BasicAuthentication, configuration.username, configuration.password
+          builder.use Faraday::Request::Authorization, :basic, configuration.username, configuration.password
         end
 
         builder.use Faraday::Response::Logger if ENV['DEBUG']
+        builder.use Faraday::Response::Json
```

Comment:
It looks like this moved out of the `unless options[:raw]` block, is that intentional?